packages/core/chrome/core-chrome-browser-internal/src/chrome_service.test.tsx

```diff
@@ -200,22 +200,6 @@ describe('start', () => {
       expect(shallow(React.createElement(() => chrome.getHeaderComponent()))).toBeDefined();
     });
 
-    it('renders the default project side navigation', async () => {
```

Comment:
There is no default project navigation. Each serverless project has to provide the nav component